src/iconsManifest/languages.ts

```diff
@@ -189,6 +189,7 @@ export const languages: ILanguageCollection = {
   latino: { ids: 'latino', defaultExtension: 'lat' },
   less: { ids: 'less', defaultExtension: 'less' },
   lex: { ids: 'lex', defaultExtension: 'flex' },
+  lilypond: { ids: 'lilypond', defaultExtension: 'ly' },
```

Comment:
There are two extensions, wasn't sure which one to put here.

Reply:
I guess it's ok. It doesn't really matter that much as it will get the extensions declared in the language extension.